src/RedisResourceManager.php

```diff
@@ -713,4 +768,25 @@ private function getRedisInfo(RedisResource $redis): array
 
         return $info;
     }
+
+    /**
+     * Set redis user
+     *
+     * @param string $id
+     * @param string $user
+     * @return RedisResourceManager
+     */
+    public function setUser($id, $user)
```

Comment:
Since this is a new method, we can add argument type here.
```suggestion
    public function setUser(string $id, string $user): void
```

I'd also remove `return $this;` so we can introduce `: void` return type.